icons/variant-calling/variant-calling-icon.svg

```diff
@@ -0,0 +1,221 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
this icon is subtly different than it's png counterpart of the same name

Reply:
Re-uploaded the matching png